utils/docker/Dockerfile.centos.7

```diff
@@ -164,8 +197,10 @@ enabled=1\n\
 gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;       \
         done;                                                                 \
         echo "Installing: $QUICKBUILD_DEPS";                                  \
+        dnf clean all;                                                        \
```

Comment:
Why is this needed?

Reply:
If someone had a reliability issue it's because a previous command wasn't doing a clean afterwards, so we should ensure that we check for that.  Please do not also clean at the start of RUN commands.